templates/taxi/driver_detail.html

```diff
@@ -9,6 +9,8 @@ <h1>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
   <p><strong>License number:</strong> {{ driver.license_number }}</p>
   <p><strong>Is staff:</strong> {{ driver.is_staff }}</p>
+  <p><a href="{% url "taxi:driver-update" pk=driver.id %}">Update</a></p>
+  <p><a href={% url "taxi:driver-delete" %}>Delete</a></p>
```

Comment:
The URL tag for 'driver-delete' is missing the `pk` parameter. It should be `{% url "taxi:driver-delete" pk=driver.id %}` to correctly pass the driver's primary key.